gapis/api/gles/api/state_queries.api

```diff
@@ -1006,6 +1006,10 @@ sub void GetStateVariable!T(GLenum name, bool isIndexed, GLuint index, T* v) {
     case GL_MAX_VERTEX_UNIFORM_VECTORS: {
       v[0] = as!T(ctx.Constants.MaxVertexUniformVectors)
     }
+    @if(Extension.GL_OVR_multiview)
+    case GL_MAX_VIEWS_OVR: {
+      write(v[0:1])
```

Comment:
If we don't have a constant for this that can be assigned out, please add a TODO to add one.

Reply:
I've added the constant. PTAL.